nix/cardano-services/deployments/backend-ingress.nix

```diff
@@ -68,7 +68,7 @@
             ++ lib.optionals config.providers.stake-pool-provider.enabled [
               {
                 pathType = "Prefix";
-                path = "/v${values.cardano-services.versions.stakePool}/stake-pool";
+                path = "/v${lib.last (lib.sort lib.versionOlder values.cardano-services.versions.stakePool)}/stake-pool";
```

Comment:
@gytis-ivaskevicius should `max(version)` be used here? Or all of them?

Reply:
Ah, no, `max(version)` was pseudocode, I called this construct (`lib.last (lib.sort lib.versionOlder`) `max` :)

Reply:
Ah, im okay with either